cpp/src/binaryop/compiled/binary_ops.cu

```diff
@@ -43,7 +43,7 @@ struct scalar_as_column_device_view {
   template <typename T, std::enable_if_t<(is_fixed_width<T>())>* = nullptr>
   return_type operator()(scalar const& s,
                          rmm::cuda_stream_view stream,
-                         rmm::mr::device_memory_resource* mr)
+                         rmm::mr::device_memory_resource*)
```

Comment:
It's unfortunate that we need to retain an unnamed parameter in the signature because it's only used by certain specializations :( I guess there's no way around that incongruity because of nested types like string.